docs/http/http-metrics.md

```diff
@@ -2,13 +2,16 @@
 linkTitle: Metrics
 --->
 
+<!--- comments between \<!-- semconv ... --\> and  endsemconv  are generated by YAML and shouldn't 
+be edited in this file. For more information, see
+https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#yaml-to-markdown
+--->
+
```

Comment:
I'd vote against adding a one-off comment manually here:

```suggestion
```

And I'm not sure we want the tooling to add such a comment either. It'll just bloat the files unnecessarily, without guaranteeing that folks will notice this comment at the start of the file. IMHO, having such knowledge is part of the learning curve of any contributor to this repo. That being said, if we can find a practical way of making the curve less steep, I'm all for it.